src/app.rs

```diff
@@ -2071,11 +2071,13 @@ impl<T: Frontend> App<T> {
         self.open_prompt(
             PromptConfig {
                 on_enter: DispatchPrompt::Null,
-                items: crate::themes::themes()?
+                items: crate::themes::theme_descriptor::all()
                     .into_iter()
-                    .map(|theme| {
-                        DropdownItem::new(theme.name.to_string())
-                            .set_dispatches(Dispatches::one(Dispatch::SetTheme(theme.clone())))
+                    .enumerate()
+                    .map(|(index, theme)| {
+                        DropdownItem::new(theme.name().to_string())
+                            .set_rank(Some(Box::from([index].to_vec())))
```

Comment:
Why rank by `index` instead of `theme.name()`?

Reply:
Didn't know you could, will make the change.

Reply:
Oops sorry you're right, `set_rank` only takes usize.